src/main/resources/application.yml

```diff
@@ -41,7 +41,7 @@ spring:
       scheduler.instanceId: AUTO
       scheduler.instanceName: data-compliance-quartz
 
-elite2.api.offender.ids.iteration.threads: 10
+elite2.api.offender.ids.iteration.threads: 4
```

Comment:
Reducing thread count (should be then under the 5 per second limit that Amazon currently impose)